### PR TITLE
Add two properties to set the text color of the labels (Issue #315)

### PR DESCRIPTION
**Pull request by gregoryalary**
This PR will add two properties to the component : `checkedTextColor` and `uncheckedTextColor` that change the text colors of the `checkedLabel`/`uncheckedLabel`. This PR also includes the demo and the usage (in the README.md) for these properties.
The default value remains `black` for both labels.

This would close the issue #315.

### CODE REVIEW

File: src/lib/ui-switch/ui-switch.component.ts [hunk 1]
```diff
@@ -30,11 +30,13 @@ const UI_SWITCH_CONTROL_VALUE_ACCESSOR: any = {
     [class.switch-medium]="size === 'medium'"
     [class.switch-small]="size === 'small'"
     [style.background-color]="getColor()"
-    [style.border-color]="getColor('borderColor')"
```
Comment (cmckni3) [inline src/lib/ui-switch/ui-switch.component.ts]: why was this changed?

Comment (gregoryalary) [reply src/lib/ui-switch/ui-switch.component.ts]: It was a mistake, I fixed it in the last commit.
